app/services/siade.rb

```diff
@@ -0,0 +1,78 @@
+class SiadeClientError < StandardError
+  attr_reader :code
+
+  def initialize(code, msg = 'Error with Siade client')
+    @code = code
+
+    super(msg)
+  end
+end
+
+class Siade
+  def initialize(token:)
+    @token = token
+    @token_rehash = if Rails.env.development?
+                      File.read('config/apientreprise_test_token')
+                    else
+                      token.rehash
+                    end
+  end
+
+  def entreprises(siret:)
+    siade_result("v2/entreprises/#{siret}")
+  end
+
+  def attestations_sociales(siren:)
+    siade_result("v2/attestations_sociales_acoss/#{siren}")
+  end
+
+  def attestations_fiscales(siren:)
+    siade_result("v2/attestations_fiscales_dgfip/#{siren}")
+  end
+
+  private
+
+  def siade_result(endpoint)
+    return unless endpoint && @token_rehash
+
+    result = siade_request(endpoint)
+
+    JSON.parse(result)
+  end
+
+  def siade_request(endpoint)
+    siade_url = [domain, endpoint, '?', siade_params].join
+
+    RestClient.get siade_url, siade_headers
```

Comment:
```suggestion
    RestClient.get(siade_url, siade_headers)
```

(chelou que rubocop ne crie pas ici)